addons/website/static/src/components/fields/fields.js

```diff
@@ -23,21 +24,60 @@ class PageUrlField extends Component {
     };
 
     setup() {
+        this.websiteService = useService("website");
         this.orm = useService('orm');
         this.serverUrl = `${window.location.origin}/`;
-        this.pageUrl = this.fieldURL;
 
         this.state = useState({
             redirect_old_url: false,
-            url: this.pageUrl,
+            url: this.fieldURL,
             redirect_type: '301',
+            original_url: '',
+            url_translate: '',
```

Comment:
I do not really like the fact that `url_translate`, and `original_url` are needed in the backend in case of a redirection but I did not find another way to do here.